packages/core/src/Stores/client-store.js

```diff
@@ -618,10 +623,26 @@ export default class ClientStore extends BaseStore {
         return this.dxtrade_accounts_list.some(account => account.account_type === 'real');
     }
 
+    get has_real_derivez_login() {
+        return this.derivez_accounts_list.some(account => account.account_type === 'real');
+    }
+
     hasAccountErrorInCFDList = (platform, account_type) => {
         if (!this.is_logged_in) return false;
-
-        const list = platform === CFD_PLATFORMS.MT5 ? this.mt5_login_list : this.dxtrade_accounts_list;
+        let list;
+        switch (platform) {
+            case CFD_PLATFORMS.MT5:
+                list = this.mt5_login_list;
+                break;
+            case CFD_PLATFORMS.DXTRADE:
+                list = this.dxtrade_accounts_list;
+                break;
+            case CFD_PLATFORMS.DERIVEZ:
+                list = this.derivez_accounts_list;
+                break;
+            default:
+                return false;
```

Comment:
instead of returning false, we can pass the default value of dxtrade as we were doing before
```suggestion
            case CFD_PLATFORMS.DERIVEZ:
                list = this.derivez_accounts_list;
                break;
             case CFD_PLATFORMS.DXTRADE:
              default:
                list = this.dxtrade_accounts_list;
                break;
```